src/main/java/uk/gov/hmcts/ccd/data/definition/CaseDefinitionRepository.java

```diff
@@ -24,4 +25,6 @@
 
     JurisdictionDefinition getJurisdiction(String jurisdictionId);
 
+    Optional<List<String>> getAllCaseTypesByJurisdictions(List<String> jurisdictionIds);
```

Comment:
No need to use Optional with lists. Just have it return an empty list in case of no results. The client code is going to be much simpler.

Reply:
DONE